SwiftLintDanger/ViewController.swift

```diff
@@ -9,6 +9,8 @@
 import UIKit
 
 class ViewController: UIViewController {
+    
+    var username : String?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/AkarapasWongkaew/SwiftLintDanger/blob/4e5f45acdb99cb248ac2af961662cbdeabf51909/SwiftLintDanger/ViewController.swift#L13"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.</td>
    </tr>
  </tbody>
</table>